packages/frontend/src/components/FlowSubstep/index.tsx

```diff
@@ -39,6 +39,11 @@ const validateSubstep = (substep: ISubstep, step: IStep) => {
       return true
     }
 
+    // '0' is a valid value for scheduling hour
+    if (argValue === 0) {
+      return true
+    }
+
```

Comment:
Actually, looking at the logic, the only values are invalid are `null`, `undefined` and `empty string`. Can we rewrite this logic to 
```
return argValue != null && argValue !== ''
```

Reply:
makes sense, roger that sir